src/app/app-routing.module.ts

```diff
@@ -3,13 +3,13 @@ import { RouterModule, Routes } from '@angular/router';
 import { AdminComponent } from './pages/admin/admin.component';
 import { AuthenticationComponent } from './pages/authentication/authentication.component';
 import { NotFoundComponent } from './pages/not-found/not-found.component';
+import { HomeComponent } from './pages/home/home.component';
 
 
 const routes: Routes = [
   {
     path: '',
-    redirectTo: '/home',
-    pathMatch: 'full',
+    component: HomeComponent,
```

Comment:
You already have redirection to HomeComponent in the end. You don't need to duplicate this route

Reply:
done